saltgui/static/scripts/panels/Minions.js

```diff
@@ -666,13 +666,12 @@ export class MinionsPanel extends Panel {
         txt += MinionsPanel._addCveList("salt-minion", minionVersion, minionBugs, allCveKeys);
 
         if (outcome === "Minion requires update") {
-          txt += "\nThis salt-minion (" + minionVersion + ") is older than the salt-master (" + masterVersion + ")";
+          txt += "\nThis salt-minion (" + minionVersion + ") is older than the salt-master (" + masterVersion + ")\nUpgrade minion is highly recommended!";
         } else if (outcome === "Minion newer than master") {
-          txt += "\nThis salt-minion (" + minionVersion + ") is newer than the salt-master (" + masterVersion + ")";
+          txt += "\nThis salt-minion (" + minionVersion + ") is newer than the salt-master (" + masterVersion + ")\nDowngrade minion is highly recommended!";
```

Comment:
what about "Upgrade master is highly recommended!"?

Reply:
It seems to me that you need to update the master to get new functionality and fix vulnerabilities. In this case, if the version of the master and the minion diverge, it is better to suggest downgrade the minion version